Minisat/mtl/XAlloc.h

```diff
@@ -39,7 +39,8 @@ namespace Minisat {
 class OutOfMemoryException{};
 static inline void* xrealloc(void *ptr, size_t size)
 {
-    void* mem = REALLOC_UNKNOWN(ptr, size, "Minisat::xrealloc");
+    Lib::TimeoutProtector tp;
+    void* mem = std::realloc(ptr, size);
```

Comment:
Why do we sometimes (e.g., in `Minisat/mtl/Vec.h`) use `xrealloc`/`xfree` and here `std::realloc`/`str::free`?

Reply:
So (I assume) all the `*ALLOC_UNKNOWN` calls in Minisat used to be calls to Minisat's `xrealloc` or `xfree`. Working on that assumption I reverted it - which might be bogus, but seems to be "reasonable enough". Here it's `std::realloc` because we are *in* `xrealloc`.

Reply:
Oh yes, forgive me my stupidity.